CHANGELOG.md

```diff
@@ -0,0 +1,56 @@
+# Changelog
+
+## Version 2.0.0
+
+### Changes to the SAML integration
+
+#### Defaults to the SHA-256 hash signature hash
+
+The module are now by default using the SHA-256 instead of the deprecated
+SHA-1 signature signing algorithm. SHA-1 is no longer recommended for ongoing
+use.
+
+If you wish to keep the SHA-1 algorithm, you will need to add an override
+to the `mysite/_config/saml.yml`:
+
+```
+SAMLConfiguration:
+ Security:
+   signatureAlgorithm: "http://www.w3.org/2000/09/xmldsig#rsa-sha1"
+```
+
+Otherwise the ADFS settings for the relying party trust needs to be changed
+from 'SHA-1' to 'SHA-256', see the `Set the secure hash algorithm` section
+in the [ADFS administrator guide](docs/en/adfs.md) for what setting to change.
+
+#### Absolute paths for the SP and IDP certificates and key files
+
+We have fixed an issue where you couldn't use absolute paths for the
```

Comment:
How about "An issue was fixed where absolute paths couldn't be used for certificates and keys in YAML configuration"?